wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,442 @@
+
+As part of this, an [Actor's Rolldata](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) is available as "@attributes." Useful examples:
+
+| @attribute                 | Description                   |
+| -------------------------- | ----------------------------- |
+| `@abilities.dex.mod`       | Actor's Dexterity Modifier    |
+| `@prof`                    | Actor's Proficiency Bonus     |
+| `@details.level`           | Actor's overall Level         |
+| `@classes.barbarian.levels` | Actor's Barbarian Class Level |
+
+> [!Note]
+> When using formulas in an Active Effect Value, the actor sheet display that corresponds to the changed value will not display the evaluated formula, but it will be applied when rolled.
+> E.g. When adding `+@abilities.cha.mod` to `system.bonuses.abilities.save` to simulate a Paladin's Aura of Protection, the actor sheet will not display that bonus applied to saving throws. The bonus will be present when the saving throw is rolled.
+
+| Change Mode | Description |
+|------------ | ------------|
+| Add         | Adds the provided value to a number. This can be used to both add and subtract from a particular value by specifying `+1` or `-1` as the value to add. |
+| Multiply    | Multiplies the defined attribute by the numeric value in the Effect Value field.|
+| Override    | Replaces the defined attribute with the value provided in the Effect Value field.|
+| Downgrade   | Reduces the defined attribute only in cases where the current value of that attribute would be greater than value specified in the Effect Value field.|
+| Upgrade     | Increases the defined attribute only in cases where the current value of that attribute would be less than value specified in the Effect Value field. |
+| Custom      | The Custom change mode applies logic defined by a game system or add-on module. The dnd5e system does not utilize the Custom Change Mode|
+
+> [!important]
+> When using the `ADD` change mode, it is always recommended to include the `+` in the Effect Value, this will ensure that if you have multiple effects targeting the same Attribute, they will not be concatenated (e.g. two effects that `ADD | +1` will result in a bonus of `+1+1`, whereas two effects that `ADD | 1` will result in a bonus of `+11`.
+
+## Commonly Desired Effect Examples
+
+### Abilities
+
+```
+system.abilities.[abbreviation].value
+                                bonuses.check
+                                        save
+```
+
+
+> <details>
+> <summary>Ability Abbreviations</summary>
+> 
+> | Ability      | Abbreviation |
+> | ------------ | ------------ |
+> | Strength     | `str`        |
+> | Dexterity    | `dex`        |
+> | Constitution | `con`        |
+> | Wisdom       | `wis`        |
+> | Intelligence | `int`        |
+> | Charisma     | `cha`        |
+> 
+> Source: `CONFIG.DND5E.abilities`
+>   
+> </details>
+
+
+#### Overriding an Ability Value
```

Comment:
It occurs to me that we might want to try to stick to game terms where possible as that is most likely what people are thinking of when trying to translate game rules to active effects. I'm just going to go through and flag these to hopefully make it easier if you agree.

```suggestion
#### Overriding an Ability Score
```